mote/.stylelintrc

```diff
@@ -0,0 +1,359 @@
+        - margin-bottom
+        - margin-left
+        - padding
+        - padding-top
+        - padding-right
+        - padding-bottom
+        - padding-left
+        - float
+        - clear
+        - clip
+        - columns
+        - column-gap
+        - column-fill
+        - column-rule
+        - column-span
+        - column-count
+        - column-width
+        - transform
+        - transform-box
+        - transform-origin
+        - transform-style
+        - transition
+        - transition-delay
+        - transition-duration
+        - transition-property
+        - transition-timing-function
+        - border
+        - border-top
+        - border-right
+        - border-bottom
+        - border-left
+        - border-width
+        - border-top-width
+        - border-right-width
+        - border-bottom-width
+        - border-left-width
+        - border-style
+        - border-top-style
+        - border-right-style
+        - border-bottom-style
+        - border-left-style
+        - border-radius
+        - border-top-left-radius
+        - border-top-right-radius
+        - border-bottom-left-radius
+        - border-bottom-right-radius
+        - border-color
+        - border-top-color
+        - border-right-color
+        - border-bottom-color
+        - border-left-color
+        - outline
+        - outline-color
+        - outline-offset
+        - outline-style
+        - outline-width
+        - background
+        - background-attachment
+        - background-clip
+        - background-color
+        - background-image
+        - background-repeat
+        - background-position
+        - background-size
+        - color
+        - font
+        - font-family
+        - font-size
+        - font-smoothing
+        - font-style
+        - font-variant
+        - font-weight
+        - letter-spacing
+        - line-height
+        - list-style
+        - text-align
+        - text-decoration
+        - text-indent
+        - text-overflow
+        - text-rendering
+        - text-shadow
+        - text-transform
+        - text-wrap
+        - white-space
+        - word-spacing
+        - border-collapse
+        - border-spacing
+        - box-shadow
+        - caption-side
+        - content
+        - cursor
+        - empty-cells
+        - opacity
+        - overflow
+        - quotes
+        - speak
+        - table-layout
+        - vertical-align
+        - visibility
+        - z-index
+    declaration-block-semicolon-newline-after: always
+    declaration-block-semicolon-newline-before: never-multi-line
+    declaration-block-semicolon-space-after: always-single-line
+    declaration-block-semicolon-space-before: never
+    declaration-block-single-line-max-declarations: 1
+    declaration-block-trailing-semicolon: always
+
+    # Block (http://stylelint.io/user-guide/rules/#block)
+    block-closing-brace-empty-line-before: never
+    block-closing-brace-newline-after:
+        - always
```

Comment:
this should check if its EOF since its not necessary for newlines at end of file.